cset-workflow/app/metplus_grid_stat/file/GridStat.conf

```diff
@@ -0,0 +1,88 @@
+[dir]
+FCST_BASE = {ENV[METPLUS_FCST_DIR]}
+ANA_BASE = {ENV[METPLUS_ANA_DIR]}
+OUTPUT_BASE = {ENV[CYLC_TASK_WORK_DIR]}
+MET_INSTALL_DIR = {ENV[MET_INSTALL_DIR]}
+TIME_START = {ENV[TIME_START]}
+
+[config]
+
+# Documentation for this use case can be found at
+# https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/ASCII2NC/ASCII2NC_python_embedding.html
+
+# For additional information, please see the METplus Users Guide.
+# https://metplus.readthedocs.io/en/latest/Users_Guide
+
+###
+# Processes to run
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#process-list
+###
+
+SCRUB_STAGING_DIR = False
+PROCESS_LIST = GridStat
+LOG_GRID_STAT_VERBOSITY = 4
+
+###
+# Time Info
+# LOOP_BY options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = INIT
+INIT_TIME_FMT = %Y%m%dT%H
+INIT_BEG = {TIME_START}
+INIT_END = {TIME_START}
+INIT_INCREMENT = 1H
+
+#LEAD_SEQ = 0
+LEAD_SEQ = begin_end_incr(0,6,1)
+
+###
+# File I/O
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#directory-and-filename-template-info
+###
+
+FCST_GRID_STAT_INPUT_DIR = {FCST_BASE}
+FCST_GRID_STAT_INPUT_TEMPLATE = akl_0p333km_{TIME_START}_met.nc
+
+OBS_GRID_STAT_INPUT_DIR = {ANA_BASE}
+OBS_GRID_STAT_INPUT_TEMPLATE = analysis_air_temperature_{valid?fmt=%Y%m%d%2H}.nc
```

Comment:
Hard coded input templates are not ideal. Perhaps this should be a rose edit variable?

Reply:
Agreed, to some extent... These input templates are going to be site dependent, so they can't go in a file that is not site-specific, like this one; however, this configuration file will not be used, as it will be replaced by the site-specific configurations from the cset-workflow repository. I thought in removing this file for tidy up, but I prefer to keep it for now as a template. Once we have more sites using the METPlus apps we will have a better idea on what to tidy up.